homeassistant/components/switcher_kis/__init__.py

```diff
@@ -0,0 +1,268 @@
+"""Home Assistant Switcher Component.
```

Comment:
We now use a single line: `"""Support for Switcher Kis Boiler."""`

Reply:
Fixed in last commit.